implSrc/jei/java/io/github/mattidragon/tlaapi/impl/jei/TlaApiJeiPlugin.java

```diff
@@ -242,13 +275,24 @@ public <T extends Screen> void addScreenSizeProvider(Class<T> clazz, Function<T,
         sizeProviders.add(new ScreenSizeProvider<>(clazz, provider));
     }
 
+    @SuppressWarnings("unchecked")
     @Override
     public <T extends Screen> void addExclusionZoneProvider(Class<T> clazz, Function<T, ? extends Iterable<TlaBounds>> provider) {
         if (!HandledScreen.class.isAssignableFrom(clazz)) return; // JEI only supports exclusion zones on handled screens
         //noinspection unchecked
```

Comment:
As you added the supress to the method, please remove the comment supression

Reply:
The comment suppression didn't do anything so I added it. Will get rid of it